### DISCUSSION TITLE
docker error

### DISCUSSION THREAD

**Discussion by CopterLifwig12** [community/community#82182] [Discussions] [answered]
### Select Topic Area

Question

### Body

╰─ docker                                                                                                         ─╯

The command 'docker' could not be found in this WSL 2 distro.
We recommend to activate the WSL integration in Docker Desktop settings.

For details about using Docker Desktop with WSL 2, visit:

https://docs.docker.com/go/wsl2/
help

**Comment by Terminal127** [comment, marked as answer]
check if docker daemon is running or not or if you are using docker desktop make sure that it is running in bg .. hope it helps

**Comment by Imadnajam** [comment]
Open Docker Desktop and go to settings. In the settings, there should be an option to enable WSL integration. Ensure that it is enabled for the specific WSL 2 distribution you are using.

**Comment by Akash1134** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
